content/_snippets/tx-metadata-field-table.md

```diff
@@ -0,0 +1,7 @@
+| Field                                   | Value               | Description  |
```

Comment:
Comments on this file should go to #554, not here. (This file will leave the diff after I merge that PR and rebase.)